src/components/multiuser-mode.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Setup the Networked-Aframe scene component based on query parameters
+ */
+AFRAME.registerComponent('multiuser-mode', {
+  init: function () {
+    var params = this.getUrlParams();
+
+    if (params.room === "") {
+      window.alert('Please add a room name in the URL, eg. ?room=myroom');
+    }
+
+    if (this.el.isMobile) {
+      // Mobile controls.
+      this.el.querySelector('[camera]').removeAttribute('look-controls');
+      this.el.querySelector('[camera]').setAttribute('orbit-controls', '');
+    }
+    else {
+      // Don't let replayer interfere with mobile when testing.
+      this.el.setAttribute('avatar-replayer', '');
+    }
+
+    if (params.hasOwnProperty('room') && !params.hasOwnProperty('avatar-recording')) {
+      this.el.querySelector('[local-player]').setAttribute('spawn-in-circle', {radius: 2});
+    }
+
+    var webrtc = params.hasOwnProperty('webrtc');
+    if (params.room) {
+      this.el.setAttribute('networked-scene', {
+        app: 'a-painter',
+        room: params.room,
+        signalURL: 'https://haydenlee.io/',
```

Comment:
That'd be great. Yes its just a node server. This is it: https://github.com/haydenjameslee/networked-aframe/blob/master/server/server.js

By default that server is open for anybody to use it for any application. That makes it pretty easy to cause problems if somebody uses the same "app" name as somebody else. Given the low number of users in WebVR right now this may not be worth considering right now.

Its on my roadmap to write a new server with more security, and host that as an official "networked-aframe" server that anybody can use. If mozilla/aframe wants to host that I'm all for it.